### ISSUE TITLE
There is a bug with data type conversion during install

### ISSUE THREAD

**Issue by ghost** [lukeroth/gdal#29]
```
../work/src/github.com/lukeroth/gdal/ogr.go:1252: cannot use C.GIntBig(fid) (type C.GIntBig) as type C.long in argument to _Cfunc_OGR_F_SetFID
../work/src/github.com/lukeroth/gdal/ogr.go:1343: cannot use C.GIntBig(index) (type C.GIntBig) as type C.long in argument to _Cfunc_OGR_L_SetNextByIndex
../work/src/github.com/lukeroth/gdal/ogr.go:1348: cannot use C.GIntBig(index) (type C.GIntBig) as type C.long in argument to _Cfunc_OGR_L_GetFeature
../work/src/github.com/lukeroth/gdal/ogr.go:1364: cannot use C.GIntBig(index) (type C.GIntBig) as type C.long in argument to _Cfunc_OGR_L_DeleteFeature
```

**Comment by icholy** [comment]
I'm getting a similar error
```
# github.com/lukeroth/gdal
./ogr.go:1252: cannot use _Ctype_longlong(fid) (type _Ctype_longlong) as type _Ctype_long in argument to func literal
./ogr.go:1343: cannot use _Ctype_longlong(index) (type _Ctype_longlong) as type _Ctype_long in argument to func literal
./ogr.go:1348: cannot use _Ctype_longlong(index) (type _Ctype_longlong) as type _Ctype_long in argument to func literal
./ogr.go:1364: cannot use _Ctype_longlong(index) (type _Ctype_longlong) as type _Ctype_long in argument to func literal
```

**Comment by icholy** [comment]
The problem is that my `org_api.h` files are declared as accepting `long` instead of `GIntBig`. So if I change:
``` c
OGRErr CPL_DLL OGR_F_SetFID( OGRFeatureH, long );
```
to
``` c
OGRErr CPL_DLL OGR_F_SetFID( OGRFeatureH, GIntBig );
```
It compiles fine.

**Comment by lukeroth** [comment]
Switch to using 64-bit integers for feature indices.

**Comment by icholy** [comment]
Implemented in 8522f20c5234e2f3a95e80881c92008cfa0bcd08
